File: tests/testdata/hello.cs
using System;

public class Salutation
{

    public string Greeting { get; private set; }
    public string Farewell { get; private set; }

    public Salutation()
        : this("Hello", "Goodbye")
    {
    }

    public Salutation(string greeting, string farewell)
    {
        this.Greeting = greeting;
        this.Farewell = farewell;
    }

    /// <summary>
    /// Mainline
    /// </summary>
    /// <param name="args"></param>
    /// <remarks></remarks>
    public static void Main(string[] args)
    {
        Salutation WalmartGreeter = new Salutation("Welcome to Walmart(TM)", "Thank you for shopping at Walmart(TM)");
        Salutation KlingonCommander = new Salutation("nuqneH?", "Qapla'");

        Console.WriteLine(WalmartGreeter.Greeting);
        Console.WriteLine(WalmartGreeter.Farewell);
        Console.WriteLine();
        Console.WriteLine(KlingonCommander.Greeting);
        Console.WriteLine(KlingonCommander.Farewell);
    }
}

File: tests/testdata/hello.vb
Imports System
Public Class Salutation

    Public Property Greeting As String
    Public Property Farewell As String

    Public Sub New()
        MyClass.New("Hello", "Goodbye")
    End Sub

    Public Sub New(ByVal greeting As String, ByVal farewell As String)
        Me.Greeting = greeting
        Me.Farewell = farewell
    End Sub

    ''' <summary>
    ''' Mainline
    ''' </summary>
    ''' <param name="args"></param>
    ''' <remarks></remarks>
    Public Shared Sub Main(ByVal args As String())
        Dim WalmartGreeter As New Salutation("Welcome to Walmart(TM)", "Thank you for shopping at Walmart(TM)")
        Dim KlingonCommander As New Salutation("nuqneH?", "Qapla'")

        Console.WriteLine(WalmartGreeter.Greeting)
        Console.WriteLine(WalmartGreeter.Farewell)
        Console.WriteLine()
        Console.WriteLine(KlingonCommander.Greeting)
        Console.WriteLine(KlingonCommander.Farewell)
    End Sub
End Class